examples/laz_dragndrop.html

```diff
@@ -45,23 +45,21 @@
                         view.scene.add(points);
                         points.updateMatrixWorld(true);
 
-                        var camera = view.camera.camera3D;
-
                         var lookAt = new itowns.THREE.Vector3();
                         var size = new itowns.THREE.Vector3();
                         geometry.boundingBox.getSize(size);
                         geometry.boundingBox.getCenter(lookAt);
 
-                        camera.far = Math.max(2.0 * size.length(), camera.far);
+                        cameraThree.far = Math.max(2.0 * size.length(), camera.far);
```

Comment:
var cameraThree= view.getThreeJsCamera();
This line seems to be  missing ?

Reply:
well spoted! done